docs/en/integrations/data-ingestion/clickpipes/postgres/source/rds.md

```diff
@@ -85,6 +85,13 @@ If you want to restrict traffic to your RDS instance, please add the [documented
 
 To connect to your RDS instance through a private network, you can use AWS PrivateLink. Follow our [AWS PrivateLink setup guide for ClickPipes](/knowledgebase/aws-privatelink-setup-for-clickpipes) to set up the connection.
 
+### Workarounds for RDS Proxy
+RDS Proxy does not support logical replication connections. If you have dynamic IP addresses in RDS and cannot use DNS name or a lambda, here are some alternatives:
```

Comment:
Can you add the Information about RDS Proxy in the main RDS doc as well?

Reply:
`use DNS name` -> `use a DNS name`